src/bootstrap/config.rs

```diff
@@ -1149,6 +1173,13 @@ impl Config {
             config.rust_profile_use = flags.rust_profile_use.or(rust.profile_use);
             config.rust_profile_generate = flags.rust_profile_generate.or(rust.profile_generate);
             config.download_rustc_commit = download_ci_rustc_commit(&config, rust.download_rustc);
+
+            config.rust_lto = rust
+                .lto
+                .as_deref()
+                .map(RustcLto::from_str)
+                .map(|v| v.expect("invalid value for rust.lto"))
```

Comment:
I think we can probably just unwrap or similar here -- this will end up with "invalid value for rust.lto: Invalid value for rustc LTO: ...", right?